### DISCUSSION TITLE
Comics/Mangas synced via kobo sync have a margin on the kobo reader.

### DISCUSSION THREAD

**Discussion by PixyTM** [gotson/komga#1732] [Ideas]
### Steps to reproduce

1. Setup kobo sync with a komga server
2. Sync an epub file that was converted from a cbz with Kindle Comic Converter
3. Observe the margins on the file that was synced to the kobo.

### Expected behavior

There is no margin and the images are in full-screen on the e-reader.
![PXL_20241008_231429525](https://github.com/user-attachments/assets/619e175b-242d-448b-af57-e269d88dc5fe)
(Example of a file synced via calibre-web)

### Actual behavior

There is a margin and the images are not in full-screen on the e-reader.
![PXL_20241008_225808631](https://github.com/user-attachments/assets/04624a68-cee5-4ec3-b1f4-37fe33114115)
(Example of a file synced via komga)

### Komga version

v1.14.0-master

### Operating system

Windows 10

### Installation method

Docker

### Other details

This is an issue that was happening with calibre-web before, but it has since been fixed.
Here are the various links that were related to the issue:
https://github.com/janeczku/calibre-web/issues/1514
https://github.com/janeczku/calibre-web/issues/1219
https://github.com/janeczku/calibre-web/pull/2262
It had something to do with fixed layout epubs, if I make no mistakes.

### Acknowledgements

- [X] I have searched the existing issues (open AND closed) and this is a new ticket, **NOT** a duplicate or related to another open issue.
- [X] I have written a short but informative title.
- [X] I have checked the [FAQ](https://komga.org/docs/faq).
- [X] I have updated the app to the latest version.
- [X] I will fill out all of the requested information in this form.

**Comment by gotson** [comment]
Komga already knows if an EPUB is fixed-layout or not. It seems that what is needed is to return `EPUB3FL` as the format for Kobo Sync instead of `KEPUB`.

I will need to give it a try and see how it works.

**Reply by gotson** [reply]
I've got some satisfying results by forcing the format sent to Kobo.

**Original image**, which has some prominent white borders.

![9782864973515-7](https://github.com/user-attachments/assets/8c4743a4-e1a0-4c02-9551-2f09fe9c3a10)

**Current Kobo Sync implementation (KEPUB format)**, shows the chapter at the top and page number at the bottom.

![20241009_095957](https://github.com/user-attachments/assets/3a815f20-7917-47ca-8ce7-4961d35a3d8c)

**New Kobo Sync implementation (EPUB3FL format)**, no more chapter or page number displayed, uses the screen better.

![20241009_101523](https://github.com/user-attachments/assets/90553c11-a2db-4a44-9c1b-f601e5ca89d7)

**Reply by gotson** [reply]
solved in 406274de86ee8ecf9b5f993b6619b1369e1024ee

**Reply by PixyTM** [reply]
That's amazing, thanks!

**Comment by PixyTM** [comment]
Hello, I've tested the change that you've made, and it's working! It's only not working with books that are oneshots. It still adds a margin (the old behavior) with those. Thanks again!

**Reply by gotson** [reply]
Was those existing books ? You need to force the sync for existing books, else Kobo will use the old value. There's no relationship between one shot and format.
